src/gui/processing/qgsprocessingwidgetwrapper.h

```diff
@@ -0,0 +1,356 @@
+/***************************************************************************
+                         qgsprocessingwidgetwrapper.h
```

Comment:
Same question about the filename (and the `#ifndef` / `#define` macros) considering that the class name is `QgsProcessingContextGenerator`.

Reply:
There's also `QgsAbstractProcessingParameterWidgetWrapper` and `QgsProcessingParameterWidgetFactoryInterface`, so my comment is not relevant.